react/src/index.js

```diff
@@ -130,14 +139,20 @@ class App extends Component {
                 <Route exact path="/" component={Home} />
                 <Route path="/about" component={About} />
                 <Route path="/cart" component={Cart} />
-                <Route path="/checkout" component={Checkout} />
+                <Route path="/checkout">
+                  <Checkout backend={BACKEND_URL} history={history} />
```

Comment:
Since we are no longer passing `Checkout` as a `component`, we lose out on the inclusion of `history` by default. 

- docs link: https://reactrouter.com/web/api/Route/route-props

Of the three routes I changed, this only affects the `/checkout` route.

Reply:
so you couldn't do
`<Route path="/checkout" backend={BACKEND_URL} component={Checkout} />`? Wouldn't that be less refactoring? But maybe Checkout wouldn't get access to this.props.backend that way. Not sure.

I remember having to decide or evaluate once the two different ways of doing this...I can't remember if one way caused a loss of functionality or not, but as long as the transactions are still getting full context and parameterizing correctly then we should be good. The ones with `<SentryRoute>` are different.

Reply:
Yeah let me double check if we can do it that way and report back - i tried a few different things and don't remember if that was one of them. If we can do it that way they I support it.

Reply:
@thinkocapo I double checked this: `<Route path="/checkout" component={Checkout} backend={BACKEND_URL} />`. 

From within the `Checkout` component, `this.props.backend` appears not to have been passed correctly, as it shows up as `undefined`. So as far as a path forward, we could go with the way I've done it here, or, if you have other suggestions I'm open to them.